.travis.yml

```diff
@@ -101,6 +101,15 @@ jobs:
         NO_DEPENDS=1
         GOAL="install"
         BITCOIN_CONFIG="--enable-zmq --disable-wallet --with-gui=qt5 CPPFLAGS=-DDEBUG_LOCKORDER --with-sanitizers=thread --disable-hardening --disable-asm CC=clang CXX=clang++"
+# x86_64 Linux (trusty, no depends, only system libs)
+    - stage: test
+      env: >-
+        HOST=x86_64-unknown-linux-gnu
+        DOCKER_NAME_TAG=ubuntu:14.04
+        PACKAGES="libglib2.0-dev libicu-dev libpng-dev python3-zmq qtbase5-dev qttools5-dev-tools protobuf-compiler libdbus-1-dev libharfbuzz-dev libprotobuf-dev"
+        DEP_OPTS="NO_QT=1 NO_UPNP=1 DEBUG=1 ALLOW_HOST_PACKAGES=1"
```

Comment:
This should say `NO_DEPENDS=1` and we should use the boost libs from ubuntu (E.g. https://packages.ubuntu.com/trusty/libboost-thread-dev)?

Reply:
See  #15056